mlrun/model.py

```diff
@@ -577,6 +679,103 @@ def handler_name(self):
                 return str(self.handler)
         return ""
 
+    def read_type_hints_from_inputs(self):
```

Comment:
the method name and the description aren't aligned and its a bit confusing, as at a first glance I thought it would return me the type hints while actually it overwrites the inputs param

Reply:
taking care of it

Reply:
Changed name and docs:

```python
    def extract_type_hints_from_inputs(self):
        """
        This method extracts the type hints from the inputs keys in the input dictionary. 

        As a result, after the method ran the inputs dictionary - a dictionary of parameter names as keys and paths as 
        values, will be cleared from type hints and the extracted type hints will be saved in the spec's inputs type 
        hints dictionary - a dictionary of parameter names as keys and their type hints as values. If a parameter is 
        not in the type hints dictionary, its type hint will be `mlrun.DataItem` by default.
        """
```